cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1074,6 +1074,26 @@
       <description>Security realm used for authentication</description>
   </property>
 
+  <!--
+      Security configurations for Kerberos authentication
+  -->
+  <property>
+    <name>cdap.master.kerberos.keytab</name>
+    <value></value>
+    <description>The full path to the Kerberos keytab file containing the CDAP Master's
+    credentials.</description>
+  </property>
```

Comment:
Should we default this to a sensible value? Hadoop defaults this, since there's a property to enable/disable Kerberos usage. They use (for example) `/etc/security/keytabs/nn.service.keytab` so we should use `/etc/security/keytabs/cdap.service.keytab` here.